### PR TITLE
Fix postinstall script of detox on `sh`

### PR DESCRIPTION
**Pull request by jhen0409**
Related to https://github.com/wix/detox/issues/627#issuecomment-388379155.

We can reproduce the issue by the following script:

```bash
$ sh -c "touch script.sh && chmod +x script.sh && script.sh"
# sh: script.sh: command not found
```

### CODE REVIEW

File: detox/scripts/postinstall.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh
```
Comment (rotemmiz) [inline detox/scripts/postinstall.sh]: hey @jhen0409 
Is there a reason you removed the `-e` flag ?

Comment (jhen0409) [reply detox/scripts/postinstall.sh]: It may unnecessary for the current case, but I'll add back it to be consistent. 👍

File: detox/scripts/postinstall.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh -e
```
Comment (LeoNatan) [inline detox/scripts/postinstall.sh]: Why was this changed to `sh` from `bash`? Not saying  bash is absolutely needed here, but I don't think there is a linux installation that comes without bash?

Comment (jhen0409) [reply detox/scripts/postinstall.sh]: Alpine Linux doesn't have bash by default (it is still installable), that's why it launch scripts by sh. We use it because its image size.

Comment (rotemmiz) [reply detox/scripts/postinstall.sh]: postinstall now uses node script to support Windows among Linux and macOS.
Give it a try (as soon as a new release is published), if you still have an issue, please report/submit a new PR

**Comment by rotemmiz** [comment]
postinstall now uses node script to support Windows as well as Linux and macOS.
Please check if this solves your issue (as soon as a new release is published), If you still have problems, please report an issue/submit a PR.
In the meantime, I am closing this PR.
